src/map/npc.h

```diff
@@ -86,6 +86,7 @@ struct npc_data {
 	unsigned size : 2;
 
 	struct status_data status;
+	struct status_change sc;
```

Comment:
it is needed to support clan emblems like in official (they are based on status_change)